### ISSUE TITLE
[BUG]: Filter conveyor doesn't filter when upright?

### ISSUE THREAD

**Issue by codycat** [typical-developers/oaklands-feedback#121]
### Roblox Username

crusanfrye

### What version did this issue occur in?

1.43.1

### Describe the issue in detail.

**TL;DR** When having a filter conveyor standing upright, the conveyor will act as a blocker for any item, and the items will fall through the conveyor beneath it.

**Some context:** We intended to use the filter conveyors to help gather more of one item down from others (for instance: when an ore is on top of a stone and can't fall through a conveyor because it's on top of the stone). Our idea for the system would've allowed the ore to stop at the filter conveyor that's upright, and dropping it through the conveyor beneath it or vice versa. This turned out not to work, as the ores (or any item) would hit the upright conveyor and fall through the filter conveyor beneath it, regardless of passthrough.

### Provide some images / videos of the issue in action.

[https://cdn.discordapp.com/attachments/711075839697223761/1169647685997953105/RobloxPlayerBeta_eZ0PxcDFsG.mp4?ex=65562a4b&is=6543b54b&hm=f81189f2f6064cf92df69e18822b61921fd29f978d5759db632834126a9fbc1d&](url) sorry for long link couldnt figure out how to add a file

### Before Submitting

- [X] I have confirmed that nobody else reported this issue.
- [X] The details of this report are legitimate.

**Comment by HooferDevelops** [comment]
This is intended behavior, the way it works is it sets the collision passthrough to go through any filter belt. This is so you can put multiple filters in one spot. I think there are ways you can adjust this design to make it fit your needs.

**Comment by PassiveLemon** [comment]
I'm guessing then that #117 is intentional too?

**Comment by HooferDevelops** [comment]
> I'm guessing then that #117 is intentional too?

Yes. You could get around it by creating a buffer in-between that's a normal conveyor.
